### PR TITLE
Feature: reduce in place

### PR DESCRIPTION
**Pull request by RobertPincus**
In the frequently-used scenario that  broadband flux profiles are requested (using ty_fluxes_broadband) fluxes are accumulated in the solvers, rather than after the fact. On CPUs this reduces memory use.

### CODE REVIEW

File: .github/workflows/self-hosted-ci.yml [hunk 1]
```diff
@@ -64,12 +64,15 @@ jobs:
         source compiler_modules
         export RRTMGP_ROOT=$PWD
         export FC=ftn
+        mkdir -p $PWD/tmp
```
Comment (RobertPincus) [inline .github/workflows/self-hosted-ci.yml:67]: @alexeedm Can you explain how this temporary directory is used?

File: examples/all-sky/rrtmgp_allsky.F90 [hunk 2]
```diff
@@ -338,6 +339,9 @@ program rte_rrtmgp_clouds
   !
   !!$omp parallel do firstprivate(fluxes)
   do iloop = 1, nloops
+    ! Omit the checks starting with the second iteration
+    if (iloop == 2) call rte_config_checks(logical(.false., wl))
```
Comment (RobertPincus) [inline examples/all-sky/rrtmgp_allsky.F90:343]: @alexeedm Mmmm, do we want to make this decision on the user's behalf?

Comment (alexeedm) [reply examples/all-sky/rrtmgp_allsky.F90:343]: This is an example, so all the loops are identical. If the checks pass on the first loop, they do pass on all the rest

Comment (RobertPincus) [reply examples/all-sky/rrtmgp_allsky.F90:343]: True enough... I guess you're suggesting that timing with the checks off is more useful information than timing with the checks on? Maybe we should note this somewhere (in the output? in the Readme?)

File: .github/workflows/self-hosted-ci.yml (hunk 1 above)
Comment (alexeedm) [reply .github/workflows/self-hosted-ci.yml:67]: NVHPC compiler uses temporary folder for compilation. On daint it ran out of space in the default tmp/ folder multiple times, according to the CI logs

File: examples/all-sky/rrtmgp_allsky.F90 (hunk 2 above)
Comment (alexeedm) [reply examples/all-sky/rrtmgp_allsky.F90:343]: Yep, without the checks we can do timings pretty accurately in this example. Your call regarding the note!

**Pull request merged**